libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -118,9 +123,10 @@ AC_WPNav::TerrainSource AC_WPNav::get_terrain_source() const
 ///
 
 /// wp_and_spline_init - initialise straight line and spline waypoint controllers
+///     speed_cms should be set to a positive value or left at zero to use the default speed
```

Comment:
I've added a default value of zero so if called with no arguments it will set the arg to zero and the default speed will be used.  I think this is essentially the same behaviour.